collector/src/main/java/io/prometheus/jmx/JmxCollector.java

```diff
@@ -221,7 +219,7 @@ private Config loadConfig(Map<String, Object> yamlConfig) throws MalformedObject
 
     }
 
-    class Receiver implements JmxScraper.MBeanReceiver {
+    static class Receiver implements JmxScraper.MBeanReceiver {
```

Comment:
Setting this static doesn't work. 

```
[ERROR] /home/afalko/jmx_exporter/collector/src/main/java/io/prometheus/jmx/JmxCollector.java:[275,12] error: non-static variable config cannot be referenced from a static context
```

I'd have to make this static too:
```
private Config config;
```

Reply:
I've made all the other fixes and rebased. I've updated the change to make config static. I normally avoid `static` as much as possible and either mock or pull nested classes out.

Reply:
Just make toSnakeAndLowerCase static. It doesn't use any fields of the class.

Reply:
Fixed, but had to move it to parent class b/c static is not allowed in inner classes.